server/drivers/singer/singer.go

```diff
@@ -55,6 +55,8 @@ type Singer struct {
 	tableNamePrefix  string
 	streamTableNames map[string]string
 
+	StreamReplication map[string]string
```

Comment:
can be unexported (streamReplication map[string]string)

Reply:
to lower case